account_operating_unit/models/account_move.py

```diff
@@ -22,16 +22,6 @@ def create(self, vals_list):
                     vals["operating_unit_id"] = move.operating_unit_id.id
         return super().create(vals_list)
 
-    @api.model
```

Comment:
why is is just removed?

Reply:
It was removed from OC in v16 (https://github.com/odoo/odoo/pull/96134) and it seems that it's not used anymore outside of reports like account.invoice.report